assets/sass/components/_calendar.scss

```diff
@@ -1,123 +1,98 @@
-$colWidth: 22px;
-$colPadding: 2px;
+.calendar__month {
+  border-top: 1px solid $govuk-border-colour;
 
-$roomHeaderWidth: 100px;
+  .calendar__day {
+    margin: 0;
+    border-bottom: 1px solid $govuk-border-colour;
 
-$calendarWidth: $roomHeaderWidth + (30 * ($colWidth + ($colPadding * 2)));
+    &:focus-within {
+      border-color: transparent;
+    }
 
-.govuk-link {
-  &--booking {
-    &:link, &:visited {
-      color: govuk-colour("black");
-      font-weight: bold;
+    #calendar-key & {
+      padding: govuk-spacing(2) 0;
     }
   }
 
-  &--overbooking {
-    &:link, &:visited {
-      color: govuk-colour("white");
+  .calendar__link {
+    display: flex;
+    flex-direction: row;
+    gap: govuk-spacing(4);
+    color: $govuk-text-colour;
+    padding: govuk-spacing(2) 0;
+    text-decoration: none;
+
+    &:hover {
+      background: govuk-colour('blue');
+      color: govuk-colour('white');
       font-weight: bold;
```

Comment:
This follows the design - but IMHO, it's bad form to change the font weight on hover as it makes the text 'twitch' as the mouse moves over it. Can Stevie be swayed?

Reply:
I did notice this -- I'll check with Stevie!

Reply:
Stevie happy to go with same (normal) font weight on hover. Works!